api/src/test/java/org/openmrs/util/OpenmrsUtilUnitTest.java

```diff
@@ -137,4 +147,128 @@ public void join_shouldReturnListElementsJoinedBySeparatorIfGivenSeparatorIsNull
 
 		assertThat(OpenmrsUtil.join(moduleList, null), is("module1nullmodule2"));
 	}
+
+	@Test
+	public void isStringInArray_shouldPassIfStringIsInArray() {
+
+		String[] array = new String[]{"element1","element2","element3"};
+
+		assertTrue(OpenmrsUtil.isStringInArray("element1", array));
+	}
+
+	@Test
+	public void isInNormalNumericRange_shouldNotAllowNullValuesAsLowerAndHigherNormal() {
```

Comment:
please rename `isInNormalNumericRange_shouldNotAllowNullValuesAsLowerAndHigherNormal`
as null is allowed, the method doesnt throw an exception it just returns false. and specify the case you test which is `HiNormal`

`isInNormalNumericRange_shouldReturnFalseIfHiNormalIsNull`